src/tests.rs

```diff
@@ -1087,3 +1081,89 @@ fn mock_simple_delta_to_too_weak_full_audit() {
     let stdout = String::from_utf8(stdout).unwrap();
     insta::assert_snapshot!("mock-simple-delta-to-too-weak-full-audit", stdout);
 }
+
+// TESTING BACKLOG:
+//
```

Comment:
This part was a real relief to write out, but also oh my god there's so many more tests to write and so many weird corner cases aaaaaa